README.rst

```diff
@@ -24,7 +24,7 @@ Installation
 
 Install with setup.py::
```

Comment:
Thanks! mind deleting this line as well? It'll be out of date (and not particularly helpful.